### PR TITLE
add timezone value

### PR DESCRIPTION
**Pull request by hutnikau**

### CODE REVIEW

File: views/js/controller/runtime/testRunner.js [hunk 1]
```diff
@@ -372,7 +372,10 @@ define([
              */
             killItemSession : function (callback) {
                 testMetaData.addData({
-                    'ITEM' : {'ITEM_END_TIME_CLIENT' : Date.now() / 1000}
+                    'ITEM' : {
+                        'ITEM_END_TIME_CLIENT' : Date.now() / 1000,
+                        'ITEM_TIMEZONE' : moment().utcOffset(moment().utcOffset()).format('Z') + ' ' + /\((.*)\)/.exec(new Date().toString())[1]
+                    }
```
Comment (krampstudio) [inline views/js/controller/runtime/testRunner.js]: Do we really need the timezone abbreviation ? The form  `+02:00` or `+02` is ISO-8601

Comment (hutnikau) [inline views/js/controller/runtime/testRunner.js]: Thomas asked to include timezone name to this value. We are using `hh:mm` format in the ODS reports.

Comment (krampstudio) [inline views/js/controller/runtime/testRunner.js]: After checking, you cannot use new Date.toString to extract the TZ abbreviation. It's not consistently implemented accross browsers, for example IE11 write it as `(Pacific Dailight Time)` and Opera just ignore it and have only GMT+0200. 

The ODS isn't the subject of the TestRunner itself, and with UTC offset you can retrieve the abrreviation server side more consistently

Comment (hutnikau) [inline views/js/controller/runtime/testRunner.js]: Timezone name removed.

**Pull request merged**